introspection-engine/connectors/sql-introspection-connector/src/calculate_datamodel.rs

```diff
@@ -61,6 +122,125 @@ pub fn calculate_model(schema: &SqlSchema) -> SqlIntrospectionResult<Datamodel>
+
+                        let field_type = FieldType::Relation(RelationInfo {
+                            name: "".to_string(),
+                            to: model.name.clone(),
+                            to_fields: vec![],
+                            on_delete: OnDeleteStrategy::None,
+                        });
+
+                        let arity = match relation_field.arity {
+                            FieldArity::Required | FieldArity::Optional => FieldArity::List,
+                            FieldArity::List => FieldArity::Optional,
+                        };
+
+                        let inflector = prisma_inflector::default();
+
+                        let name = match arity {
+                            FieldArity::List => inflector.pluralize(&model.name).camel_case(), // pluralize
+                            FieldArity::Optional => model.name.clone().camel_case(),
+                            FieldArity::Required => model.name.clone().camel_case(),
+                        };
+
+                        let field = Field {
+                            name,
+                            arity,
+                            field_type,
+                            database_name: None,
+                            default_value: None,
+                            is_unique: false,
+                            id_info: None,
+                            scalar_list_strategy: None,
+                            documentation: None,
+                            is_generated: false,
+                            is_updated_at: false,
+                        };
+
+                        fields_to_be_added.push((other_model.name.clone(), field));
+                    }
+                }
+                _ => {}
+            }
+        }
+    }
+
+    // add many to many relation fields
+    for table in schema
+        .tables
+        .iter()
+        .filter(|table| is_many_to_many_relation_table(&table))
+    {
+        // use foreign keys to find tables that are linked to??
+
+        let first = table.foreign_keys.get(0);
+        let second = table.foreign_keys.get(1);
+
+        match (first, second) {
+            (Some(f), Some(s)) => {
+                fields_to_be_added.push((
+                    s.referenced_table.clone(),
+                    create_many_to_many_field(f, &table.name),
+                ));
+                fields_to_be_added.push((
+                    f.referenced_table.clone(),
+                    create_many_to_many_field(s, &table.name),
+                ));
+            }
+            (_, _) => (),
+        }
+    }
+
+
+    // add scalar lists fields
+    for table in schema.tables.iter().filter(|table| is_scalar_list_table(&table)) {
+        let name = table.name.split('_').nth(1).unwrap();
+        let model = table.name.split('_').nth(0).unwrap();
+
+        let field_type = calculate_field_type(&table.columns.iter().find(|c| c.name == "value").unwrap(), &table);
```

Comment:
Do we always have a column called `value`? Again a place where we get a runtime error due to the `unwrap`.

Reply:
Yes, the is_prisma_scalar_list_table function verifies before this is being called